### PR TITLE
Add Cura Voron V0 Instructions

### PR DESCRIPTION
**Pull request by jgehrig**
Adds a set of installation instructions for the Voron V0 + Cura. A profile for the V0 is not include with Cura.

The settings are based on the Voron V2 profiles included with Cura. This is the configuration I've been using to slice parts with my V0 printer.

Perhaps it is also worth submitting a Pull Request in the Cura repository to include these profiles in future releases?

Let me know if there is anything that should be addressed, I am happy to make changes.

Thanks!

### CODE REVIEW

File: slicer_configurations/README.md [hunk 1]
```diff
@@ -23,11 +23,12 @@ like so:
 
 ---
 
-| Creator | Config title | Description | V0 | V1.8 | V2.4 | VSW |
-| --- | --- | --- | --- | --- | --- | --- |
-| hartk1213 | [PrusaSlicer V0 Bed and Textures](./PrusaSlicer/hartk1213/V0) | Bed STL and textures for Prusa Slicer | :heavy_check_mark: | :x: | :x: | :x: |
-| hijax_pl_ | [Simplify3D](./Simplify3D/hijax_pl/beds) | Bed images for Simplify3D | :x: | :x: | :heavy_check_mark: | :x: |
-| hoangnam123 | [SuperSlicer v2.4 Bed](./SuperSlicer/hoangnam123) | Bed STL for SuperSlicer | :x: | :x: | :heavy_check_mark: | :x: |
-| koonweee | [SuperSlicer v1.8 Bed](./SuperSlicer/koonweee/beds) | Bed STL for SuperSlicer | :x: | :heavy_check_mark: | :x: | :x: |
+| Creator     | Config title                                                  | Description                            | V0                 | V1.8               | V2.4               | VSW |
```
Comment (jgehrig) [inline slicer_configurations/README.md:26]: This table cleanup not strictly necessary.

I can revert these changes if the noisy diff is obnoxious.

**Comment by jgehrig** [comment]
The file `slicer_configurations/Cura/jgehrig/V0/cura-config-folder/meshes/V0_120mm_Bed.stl` is a copy of:
https://github.com/jgehrig/VoronUsers/blob/master/slicer_configurations/PrusaSlicer/hartk1213/V0/Bed_Shape/Model/V0_120mm_Bed.stl

Perhaps there is a way to symlink or use a Git LFS link to the same file? That would save ~40MB of increased repository size...

File: slicer_configurations/README.md (hunk 1 above)
Comment (KiloQubit) [reply slicer_configurations/README.md:26]: Yeah, don't do that, the next person that adds an entry is not going to care about alignment, and the table will be out of alignment again anyway.  As much as I personally would prefer the table to be aligned, it's not going to remain that way.

Comment (jgehrig) [reply slicer_configurations/README.md]: Fixed. Yeah, I thought the same thing after writing this.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your PR!  Please see the requested changes below.

**Comment by jgehrig** [comment]
@KiloQubit Thanks for the review!

Comments resolved. Let me know if you'd like any further modification.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for the update.  There's one change that was not implemented below (startup g-code line number)

File: slicer_configurations/Cura/jgehrig/V0/README.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+G92 E0.0             ; reset extruder
+G1 E4 F500.0         ; pre-purge prime LENGTH SHOULD MATCH YOUR PRINT_END RETRACT
+G1 X80 E10.0 F500.0  ; intro line 1
+G1 Y0.3              ; move in a little
+G1 X40 E10.0 F500.0  ; second line
+G92 E0.0             ; reset extruder
+G1 Z2.0              ; move nozzle to prevent scratch
+print_start
+```
+**NOTE:** The pre-purge prime (line 4) should match the length of retract in `PRINT_END` in
```
[the remark this answers is not in the document]
Comment (jgehrig) [reply slicer_configurations/Cura/jgehrig/V0/README.md]: Whoops!

**Comment by jgehrig** [comment]
@KiloQubit

Updated: `line 4` -> `line 5`

**Review by KiloQubit — APPROVED** [review]
Thanks for the update!

**Pull request merged**